packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -1042,14 +1050,45 @@ async function executeWorkflow(
 	};
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function sendMessageToUI(source: string, messages: any[]) {
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-return */
+export async function sendMessageToUI(source: string, messages: any[]): Promise<void> {
```

Comment:
We should not be disabling linting for entire code blocks. we have enough tech-debt already, this only increases the tech-debt.

Reply:
This is just WIP and also a wrong approach, just reverted it

Reply:
ah. can you please then mark the PR as `in draft` ?